python/torch_mlir_e2e_test/test_suite/conv.py

```diff
@@ -12,6 +12,7 @@
 
 
 class Conv2dNoPaddingModule(torch.nn.Module):
+
```

Comment:
Can you make all the style changes in this file in a separate PR? It makes it hard to know which changes are truly related to this PR

Reply:
Sure.

Reply:
Done.